standalone/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
```

Comment:
```suggestion
#!/bin/sh
```

IIRC, existing scripts in this repo use /bin/sh, let's make it consistent. Same below